dist/less/saas-list.less

```diff
@@ -26,10 +32,12 @@
       @media (min-width: @screen-sm-min) {
         border-bottom: none;
         flex: 0 1 auto;
```

Comment:
If i take out the min-width, the item shrink and fit so it doesn't limit the number per row correctly.